frontend/lib/src/components/elements/ExceptionElement/ExceptionElement.tsx

```diff
@@ -84,7 +86,7 @@ function ExceptionMessage({
 function StackTrace({ stackTrace }: Readonly<StackTraceProps>): ReactElement {
   // Build the stack trace display, if we got a stack trace.
   return (
-    <>
+    <div>
```

Comment:
Whats the reason why this needs to be changed to a `div`?

Reply:
Oh that was only an oversight, fixed.